xrpl4j-model/src/main/java/org/xrpl/xrpl4j/model/client/serverinfo/ServerInfo.java

```diff
@@ -0,0 +1,281 @@
+   */
+  @Value.Derived
+  @JsonIgnore
+  default boolean isLedgerInCompleteLedgers(final UnsignedLong ledgerIndex) {
+    return this.completeLedgers().stream()
+      .anyMatch(range -> range.contains(ledgerIndex));
+  }
+
+  /**
+   * The number of times (since starting up) that this server has had over 250 transactions waiting to be processed at
+   * once. A large number here may mean that your server is unable to handle the transaction load of the XRP Ledger
+   * network. For detailed recommendations of future-proof server specifications, see
+   * <a href="https://xrpl.org/capacity-planning.html">Capacity Planning</a>.
+   *
+   * @return An optionally-present {@link String}.
+   */
+  @JsonProperty("jq_trans_overflow")
+  Optional<String> jqTransOverflow();
+
+  /**
+   * (Admin only) Detailed information about the current load state of the server.
+   *
+   * @return An optionally-present {@link ServerInfoLoad}.
+   */
+  @JsonProperty("load")
+  Optional<ServerInfoLoad> load();
+
+  /**
+   * The load-scaled open ledger transaction cost the server is currently enforcing, as a multiplier on the base
+   * transaction cost. For example, at 1000 load factor and a reference transaction cost of 10 drops of XRP, the
+   * load-scaled transaction cost is 10,000 drops (0.01 XRP). The load factor is determined by the highest of the
+   * individual server's load factor, the cluster's load factor, the open ledger cost and the overall network's load
+   * factor. Per xrpl.org docs, this field is optionally present in any server response and may be omitted because it
+   * is only returned when requested via an admin host/port.
+   *
+   * @return An optionally-present {@link BigDecimal} representing the load factor.
+   */
+  @JsonProperty("load_factor")
+  Optional<BigDecimal> loadFactor();
+
+  /**
+   * Current multiplier to the transaction cost based on load to this server.  Per xrpl.org docs, this field is
+   * optionally present in any server response and may be omitted.
+   *
+   * @return An optionally-present {@link BigDecimal} representing the local load factor.
+   */
+  @JsonProperty("load_factor_local")
+  Optional<BigDecimal> loadFactorLocal();
+
+  /**
+   * Current multiplier to the transaction cost being used by the rest of the network (estimated from other servers'
+   * reported load values). Per xrpl.org docs, this field is optionally present in any server response and may be
+   * omitted.
+   *
+   * @return An optionally-present {@link BigDecimal} representing the network load factor.
+   */
+  @JsonProperty("load_factor_net")
+  Optional<BigDecimal> loadFactorNet();
+
+  /**
+   * Current multiplier to the transaction cost based on load to servers in this cluster. Per xrpl.org docs, this field
+   * is optionally present in any server response and may be omitted.
+   *
+   * @return An optionally-present {@link BigDecimal} representing the cluster load factor.
+   */
+  @JsonProperty("load_factor_cluster")
```

Comment:
not present in clio

Reply:
Resolved via 86d12e7431f35cef1e1dcbdb28df21ed5c19a0f8